services/cad-service/grpc_server.py

```
@@ -0,0 +1,93 @@
+"""gRPC server exposing CAD operations."""
+from __future__ import annotations
+
+from typing import AsyncIterator
```

Ruff (0.14.1)

Import from `collections.abc` instead: `AsyncIterator`

Import from `collections.abc`

(UP035)